backend/plugins/operation_api/src/modules/tasks/graphql/resolvers/mutations/task.ts

```
@@ -0,0 +1,566 @@
+    //       } has updated`,
+    //       receivers: notificationDoc?.item?.assignedUserIds,
+    //       data: {
+    //         type,
+    //         id: _id
+    //       }
+    //     }
+    //   });
+    // }
+
+    // exclude [null]
+    if (doc.tagIds && doc.tagIds.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)